src/main/java/org/torproject/onionoo/util/DomainNameSystem.java

```diff
@@ -0,0 +1,62 @@
+package org.torproject.onionoo.util;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Hashtable;
+import java.util.Set;
+import java.util.TreeSet;
+
+import javax.naming.Context;
+import javax.naming.NamingEnumeration;
+import javax.naming.NamingException;
+import javax.naming.directory.Attribute;
+import javax.naming.directory.Attributes;
+import javax.naming.directory.DirContext;
+import javax.naming.directory.InitialDirContext;
+
+public class DomainNameSystem {
+
+  private static DirContext dnsContext;
+  private static Logger log = LoggerFactory.getLogger(DomainNameSystem.class);
+
+  static {
+    Hashtable<String, String> envProps = new Hashtable<String, String>();
+    envProps.put(Context.INITIAL_CONTEXT_FACTORY,
+            "com.sun.jndi.dns.DnsContextFactory");
```

Comment:
Use 4 spaces for indenting here. Run `ant checks` to check things like this. (Not 100% sure whether this shows up there, I didn't try.) There are more places where more whitespace is used than necessary, and I'm not marking any subsequent places.